tests/parse.spec.js

```diff
@@ -232,4 +232,54 @@ describe('Parse', function() {
     ]);
   });
 
+  it('should handle unknown escape sequences inside a value', function() {
```

Comment:
IMHO we should also have a test on a string containing multiple unknown escape sequences